Assets/Scripts/Menu/MainMenu.cs

```diff
@@ -26,9 +27,13 @@ void Start()
         ModeButtonRender();
         ResolutionButton.text = resolutions[currentResolution].width + "x" + resolutions[currentResolution].height;
         // Reseting GUI and Game Manager
-        if (SceneManager.GetActiveScene().name == "MainMenu" && DontDestroy.Instance != null)
+        if (DontDestroy.Instance != null)
             DontDestroy.Instance.menu();
     }
+    private void Update()
+    {
+            ContinueButtonVisibility();
```

Comment:
Do it needs to be in Update?
EveryFrame we Are Updating it, instead maybe we should use it on the "Start" function and in game when user click "Escape" button?